README.md

```diff
@@ -261,6 +345,163 @@ class MyChain extends SetupChain {
 }
 ```
 
+## Usage
+
+In your test suites you can instantiate a chain instance to perform a series of async tasks
+while storing the result in a single object. Actions can do anything from returning
+random data, inserting records into a data store, to sending log entries to a remote parser. We recommend calling the code libraries the same as production would. For
+example, if you had a `organization-create.js` and a `user-create.js`, they could
+be trivially wired up to be actions in the chain:
+
+```javascript
+const state = await new MyChain()
+  .createOrganization()
+  .set('!random:5', 'user_id')
+  .createUser({id: '#user_id'})
+  .execute()
```

Comment:
this is a great example

I think your `set` should be `.set('user_id', '!random:5')`

Reply:
doh!  I'm so used to `label` as the second param.  Good catch.